mlrun/api/api/endpoints/projects.py

```diff
@@ -1,53 +1,45 @@
-from operator import attrgetter
 from http import HTTPStatus
 
 from fastapi import APIRouter, Depends, Response
 from sqlalchemy.orm import Session
 
 from mlrun.api import schemas
 from mlrun.api.api import deps
-from mlrun.api.api.utils import log_and_raise
-from mlrun.api.db.sqldb.helpers import to_dict as db2dict
 from mlrun.api.utils.singletons.db import get_db
 
 router = APIRouter()
 
 
 # curl -d '{"name": "p1", "description": "desc", "users": ["u1", "u2"]}' http://localhost:8080/project
+# TODO: remove the wrong route (/project) before 0.6.0 is out after UI change to new route
 @router.post("/project")
-def add_project(
+@router.post("/projects", response_model=schemas.ProjectOutput)
+def create_project(
     project: schemas.ProjectCreate, db_session: Session = Depends(deps.get_db_session)
```

Comment:
I'm having 2nd thoughts on our decision to not send everything to all consumers and only rely on periodic sync - seems to me that we can just replace the calls here in the endpoint and route them to the `ProjectsManager` instead of directly to the DB. This way we don't need to rely on the manager to periodically sync everything - if there's an action initiated by the client, then we simply send to the manager and it will perform the actions on all the consumers.
As a matter of fact, unless I'm missing something, currently the `ProjectsManager` APIs are not used at all, other than ensure_project which is called in the SQL DB code (and of course periodic sync, which doesn't take care of delete/update). I think this is where they should be called.
Also, see my comment below - `delete` must go to the manager.

Reply:
definitely, this is not intended choice, just forgot to update API calls to use the manager